docs/core_docs/docs/integrations/platforms/google.mdx

```diff
@@ -69,9 +74,86 @@ const input2 = [
 const res = await visionModel.invoke(input2);
 ```
 
-The value of image_url must be a base64 encoded image (e.g., data:image/png;base64,abcd124).
+:::tip
+Click [here](/docs/integrations/chat/google_generativeai) for the `@langchain/google-genai` specific integration docs
+:::
 
-### Vertex AI
+</TabItem>
+
+<TabItem value="vertexai" label="VertexAI" default>
+<IntegrationInstallTooltip></IntegrationInstallTooltip>
+
+```bash npm2yarn
+npm install @langchain/google-vertexai
+```
+
+Then, you'll need to add your service account credentials, either directly as a `GOOGLE_VERTEX_AI_WEB_CREDENTIALS` environment variable:
+
+```
+GOOGLE_VERTEX_AI_WEB_CREDENTIALS={"type":"service_account","project_id":"YOUR_PROJECT-12345",...}
+```
+
+or as a file path:
+
+```
+GOOGLE_VERTEX_AI_WEB_CREDENTIALS_FILE=/path/to/your/credentials.json
+```
+
+```typescript
+import { ChatVertexAI } from "@langchain/google-vertexai";
+// Or, if using the web entrypoint:
+// import { ChatVertexAI } from "@langchain/google-vertexai-web";
+
+const model = new ChatVertexAI({
+  modelName: "gemini-1.0-pro",
+  maxOutputTokens: 2048,
+});
+
+// Batch and stream are also supported
+const res = await model.invoke([
+  [
+    "human",
+    "What would be a good company name for a company that makes colorful socks?",
+  ],
+]);
+```
+
+Gemini vision models support image inputs when providing a single human message. For example:
+
+```typescript
+const visionModel = new ChatVertexAI({
+  modelName: "gemini-pro-vision",
+  maxOutputTokens: 2048,
+});
+const image = fs.readFileSync("./hotdog.png").toString("base64");
+const input2 = [
+  new HumanMessage({
+    content: [
+      {
+        type: "text",
+        text: "Describe the following image.",
+      },
+      {
+        type: "image_url",
+        image_url: `data:image/png;base64,${image}`,
+      },
+    ],
+  }),
+];
+
+const res = await visionModel.invoke(input2);
+```
+
+:::tip
+Click [here](/docs/integrations/chat/google_vertex_ai) for the `@langchain/google-vertexai` specific integration docs
+:::
+
+</TabItem>
+</Tabs>
+
+The value of `image_url` must be a base64 encoded image (e.g., `data:image/png;base64,abcd124`).
+
+### Vertex AI (Legacy)
```

Comment:
Would just remove this from the platform page

Reply:
Removed everything except the tip cards linking to legacy docs